package.json

```diff
@@ -31,5 +31,6 @@
   "packageManager": "yarn@1.22.19",
   "lint-staged": {
     "*.{ts,tsx,js,json,css,md}": "prettier --write"
-  }
+  },
+  "prettier": {}
```

Comment:
I have a setting in my IDE that requires a Prettier config to reformat files so I've added this. I'm using this setting because I don't want to reformat repos that don't use Prettier at all